stroeer/web/bulletin/v1/bulletin.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package stroeer.web.bulletin.v1;
+
+import "stroeer/web/article/v1/web_article.proto";
+
+option go_package = "github.com/stroeer/tapir/go/stroeer/web/bulletin/v1;bulletin";
+option java_multiple_files = true;
+option java_package = "de.stroeer.web.bulletin.v1";
+
+message Bulletin {
```

Comment:
ein (optionaler) identifier fehlt noch, oder?

Reply:
für die fixed bulletins

Reply:
wir verzichten auf IDs. 
die fixed_bulletins landen rechts in der leiste
die dynamic_bulletins landen links in der leiste
reihen folge wie vom backend
`if (label.toLowerCase() == "live")` then render red live bubble